changelog/unreleased/pull-2671

```diff
@@ -0,0 +1,5 @@
+Feature: Addition of --min-packsize and --file-read-concurrency
+
+In order to tune restic for large installs we need to be able to configure --min-packsize and --file-read-concurrency. In version 0.9.6 these numbers are hard coded, this pull request allows you to adjust these.
```

Comment:
The changelog usually doesn't refer explicitly to previous versions. The description should also be written in past tense. There's also no need to mention that this was change via a pull request. (Sorry for the nitpicking)